src/config/templates.toml

```diff
@@ -12,7 +12,9 @@ label(if(current_working_copy, "working_copy"),
     separate(" ",
       if(divergent,
         label("divergent", format_short_change_id(change_id) ++ "??"),
```

Comment:
I actually realized there is a subtle minor bug here: a commit could have a divergent change id and yet be obsolete.

It's easy to fix it here. I'm not sure if the definition of `divergent` condition should be changed, I'll probably just add a comment there for now.

I'll also see if it's not too hard to write a test. Since the bug is so minor, I might leave it for another PR.

Reply:
Actually, I think it's simpler (in terms of fixing and testing) to treat this as a bug in the "divergent" condition as opposed to this template. This also makes sense as "divergent" is a property of a commit here (not of a change id).

Reply:
I'm now thinking these should be written in white as

```
xyzzy?? hidden (normal colored email, date, etc)
```

since it makes sense to print divergent change ids differently.

Reply:
Resolved in a new commit.